app/controllers/posts_controller.rb

```diff
@@ -31,4 +31,16 @@ def create
       end
     end
   end
+
```

Comment:
- [ ] Mark your Posts controller class actions with CanCan load authorization validation by adding `load_and_authorize_resource` at the top of the class. Kindly ensue to improve upon this aspect